optuna/storages/_rdb/models.py

```diff
@@ -497,7 +496,7 @@ class TrialIntermediateValueType(enum.Enum):
     def intermediate_value_to_stored_repr(
         cls,
         value: float,
```

Comment:
```suggestion
        cls, value: float
```

Reply:
The changes have been made as suggested.